src/dinit.cc

```diff
@@ -453,6 +471,90 @@ static int process_commandline_arg(char **argv, int argc, int &i, options &opts)
     return 0;
 }
 
+#if SUPPORT_SELINUX
+// Load the system SELinux policy and transition ourselves to it. When successful,
```

Comment:
Line width should extend to (just under) 100 characters, please see CODE-STYLE. This comment wraps at just over 80 characters.